torchvision/ops/deform_conv.py

```diff
@@ -60,8 +60,8 @@ def deform_conv2d(
         >>> # returns
         >>>  torch.Size([4, 5, 8, 8])
     """
-
-    _log_api_usage_once("ops", "deform_conv2d")
+    if not torch.jit.is_scripting() and not torch.jit.is_tracing():
+        _log_api_usage_once(deform_conv2d)
```

Comment:
This looks good but we are missing the logging call on the `DeformConv2d` class constructor. Can you add it?

This is because when originally logging was added, it wasn't put both on the constructors and on the functional methods. My understanding is that we agreed to put it in both places for simplicity.

Also, should we add the call on the C++ side of the operator as well?